app/controllers/api/v1/sessions_controller.rb

```diff
@@ -9,13 +9,26 @@ def create
         self.resource = warden.authenticate!(auth_options)
         sign_in(resource_name, resource)
 
-        set_sso_response if sso_request?
-        @redirect_path ||= '/profile'
+        cookies[:token] = resource.token
+
+        @redirect_path_login ||= '/profile'
```

Comment:
You could probably remove this line, in favor of using a URL helper inline w/ the redirect_to. This _shouldn't_ really need to be memoized, unless there's another way to set it that I'm missing (e.g. some kind of before hook that _might_ set it)

If that's the case, then this needs to stay